### DISCUSSION TITLE
[Docs] Link to generate token for RESEAUX_ENERGIES_TOKEN should be updated

### DISCUSSION THREAD

**Discussion by yujia21** [electricitymaps/electricitymaps-contrib#6282] [General]
Link to add a key for RESEAUX_ENERGIES_TOKEN [here](https://github.com/electricitymaps/electricitymaps-contrib/wiki/Create-tokens#RESEAUX_ENERGIES_TOKEN) is broken, the correct page should be https://odre.opendatasoft.com/account/api-keys/ now. Can I go ahead and edit the page?

**Comment by VIKTORVAV99** [comment]
Yeah, just go ahead and edit it 🙂

Thanks in advance!
